client/blocks/reader-full-post/index.jsx

```diff
@@ -113,6 +113,10 @@ export class FullPostView extends React.Component {
 			this.attemptToSendPageView();
 		}
 
+		if ( this.props.shouldShowComments && ! prevProps.shouldShowComments ) {
```

Comment:
This is an unrelated change - the linter picked up on usage of `componentWillReceiveProps` in Reader full post. 

To test, make sure that a full post URL with `#comments` suffix still takes you to the comments section of a full post, e.g.:

http://calypso.localhost:3000/read/blogs/140522955/posts/2773#comments

Reply:
A suggestion for the future: Lazy loading images can push the scroll of the page further down, moving the comments section off-screen. I'd imagine this could be solved if we had the height information of the images.